atomics/T1550.003/T1550.003.yaml

```diff
@@ -37,3 +37,65 @@ atomic_tests:
     command: |
       #{mimikatz_exe} # kerberos::ptt #{user_name}@#{domain}
     name: command_prompt
+- name: Rubeus Kerberos Pass The Ticket
+  auto_generated_guid: a2fc4ec5-12c6-4fb4-b661-961f23f359cb
+  description: |
+    Requesting a TGT on a remote system and passing it locally to request a service ticket
```

Comment:
```suggestion
    Requesting a TGT on a remote system and retrieving it locally before requesting a service ticket with it. This is a Pass-The-Ticket attack because the TGT is obtained on the remote system, then used from a different machine (local).
    PsExec is used to execute commands on the remote system, and the "C$" admin share is used to retrieve the TGT, so the current user must have admin rights remotely and other PsExec prerequisites must be met.
```